lib/config/fileconf.go

```diff
@@ -992,6 +992,22 @@ type Auth struct {
 	// HostedPlugins configures the hosted plugins runtime.
 	// This is currently Cloud-specific.
 	HostedPlugins HostedPlugins `yaml:"hosted_plugins,omitempty"`
+
+	// PluginService is the section of the confi for configuring the plugin service.
+	PluginService PluginService `yaml:"plugin_service,omitempty"`
```

Comment:
Why is this a part of the auth config? This should be a top-level section in the config file, like other services (app, database, discovery, etc.)